libr/core/linux_heap_glibc.c

```diff
@@ -3,6 +3,7 @@
 #ifndef INCLUDE_HEAP_GLIBC_C
 #define INCLUDE_HEAP_GLIBC_C
 #define HEAP32 1
+bool main_arena_resolved = 0;
```

Comment:
use false instead of 0.

Also, don't use global variables, hold this variable inside RDebug for example